deployment-detector/src/agent.spec.ts

```diff
@@ -1,74 +1,96 @@
+import { Finding, HandleTransaction, TransactionEvent } from "forta-agent";
+import { provideHandleTransaction } from "./agent";
 import {
-  FindingType,
-  FindingSeverity,
-  Finding,
-  HandleTransaction,
-  createTransactionEvent,
-  ethers,
-} from "forta-agent";
-import agent, {
-  ERC20_TRANSFER_EVENT,
-  TETHER_ADDRESS,
-  TETHER_DECIMALS,
-} from "./agent";
+  createFindingTest,
+  FORTA_CREATE_AGENT,
+  FORTA_PROXY_ADDRESS,
+  FORTA_UPDATE_AGENT,
+  mockBotParams,
+  mockDeployerAddress,
+  mockFortaContract,
+  mockOtherFortaContract,
+  UPDATE_AGENT_EVENT,
+} from "./utils";
+import { TestTransactionEvent } from "forta-agent-tools/lib/test";
+import { createAddress } from "forta-agent-tools";
 
-describe("high tether transfer agent", () => {
+const otherDeployerAddress = createAddress("0x00");
+
+const createMockArg = (agentId: number, owner: string, metaData: string, chainId: number[]) => {
+  return {
+    agentId: agentId,
+    owner: owner,
+    metaData: metaData,
+    chainIds: chainId,
+  };
+};
```

Comment:
```suggestion
const createMockArg = (agentId: number, owner: string, chainId: number[]) => {
  return {
    agentId: agentId,
    owner: owner,
    chainIds: chainId,
  };
};
```
`metadata` no longer necessary